src/test/scala/uk/gov/hmrc/perftests/example/TrackingConsentFrontendRequests.scala

```diff
@@ -20,13 +20,13 @@ import io.gatling.core.Predef._
 import io.gatling.http.Predef._
 import io.gatling.http.request.builder.HttpRequestBuilder
 import uk.gov.hmrc.performance.conf.ServicesConfiguration
-
-import java.time.format.DateTimeFormatter
-import java.time.{ZoneOffset, ZonedDateTime}
+import scala.io.Source
+import scala.util.matching.Regex
 
 object TrackingConsentFrontendRequests extends ServicesConfiguration {
 
   private val baseUrl = baseUrlFor("tracking-consent-frontend")
+  private val defaultHmrcFrontendVersion = "5.66.0"
```

Comment:
could just throw an exception maybe, guess we won't run it locally

Reply:
There is a method to run just 1 journey, which I found useful for the purposes of testing these changes locally 🤔 Other than that uncommon scenario, we'd have no real reason to run this locally, for sure.

Reply:
even if we run it locally we should be able to get the version from the target we're testing if we don't hardcode the staging url